build/build.py

```diff
@@ -200,9 +206,12 @@ def check_bazel_version(bazel_path, min_version, max_version):
 build --define open_source_build=true
 
 # Disable enabled-by-default TensorFlow features that we don't care about.
-build --define=no_aws_support=true
-build --define=no_gcp_support=true
-build --define=no_hdfs_support=true
+build:linux --define=no_aws_support=true
```

Comment:
I suspect these can be set on windows also? They disable certain TF features and are bazel options and so should be portable. Revert this part of the change?

Reply:
Nope

```
ERROR: C:/users/cloud/_bazel_cloud/hwk42b7s/external/org_tensorflow/tensorflow/core/common_runtime/BUILD:1648:1: Illegal ambiguous match on configurable attribute "deps" in @org_tensorflow//tensorflow/core/common_runtime:core_cpu_internal:
@org_tensorflow//tensorflow:windows
@org_tensorflow//tensorflow:no_aws_support
Multiple matches are not allowed unless one is unambiguously more specialized.
ERROR: Analysis of target '//build:install_xla_in_source_tree' failed; build aborted:

C:/users/cloud/_bazel_cloud/hwk42b7s/external/org_tensorflow/tensorflow/core/common_runtime/BUILD:1648:1: Illegal ambiguous match on configurable attribute "deps" in @org_tensorflow//tensorflow/core/common_runtime:core_cpu_internal:
@org_tensorflow//tensorflow:windows
@org_tensorflow//tensorflow:no_aws_support
Multiple matches are not allowed unless one is unambiguously more specialized.
INFO: Elapsed time: 14.284s
INFO: 0 processes.
```